### DISCUSSION TITLE
how to obtain a list of individual peaks that are enriched with motifs

### DISCUSSION THREAD

**Discussion by bj-chen** [GreenleafLab/ArchR#1847] [Questions / Documentation]
Dear ArchR team,

Thank you for creating such an incredible package. 

I recently used the peakAnnoEnrichment() function to identify enriched motifs in a list of marker peaks. I wonder if there is any way that I could obtain a list of peaks that each motif is found enriched in?  Thank you!

With regards,
Bei Jun

**Comment by rcorces** [comment]
This has been answered multiple times previously.
For example https://github.com/GreenleafLab/ArchR/discussions/1644#discussioncomment-3743214
